### PR TITLE
Allow `true` as wildcard option

### PR DESCRIPTION
**Pull request by barryvdh**
Allow this value, before normalizing.

### CODE REVIEW

File: src/Asm89/Stack/CorsService.php [hunk 1]
```diff
@@ -36,16 +36,13 @@ private function normalizeOptions(array $options = array())
         );
 
         // normalize array('*') to true
-        if (in_array('*', $options['allowedOrigins'])) {
+        if ($options['allowedOrigins'] === true || in_array('*', $options['allowedOrigins'])) {
```
Comment (asm89) [inline src/Asm89/Stack/CorsService.php]: This is to ease configuration? `$options['allowedOrigins'] = true` does read a bit odd to me.

File: src/Asm89/Stack/CorsService.php [hunk 2]
```diff
@@ -36,16 +36,13 @@ private function normalizeOptions(array $options = array())
...
             $options['allowedOrigins'] = true;
         }
-        if (in_array('*', $options['allowedHeaders'])) {
+        if ($options['allowedHeaders'] === true || in_array('*', $options['allowedHeaders'])) {
             $options['allowedHeaders'] = true;
-        } else {
-            $options['allowedHeaders'] = array_map('strtolower', $options['allowedHeaders']);
```
Comment (asm89) [inline src/Asm89/Stack/CorsService.php]: Are you sure we can remove this line? We lowercase the incoming headers below.

**Comment by barryvdh** [comment]
That might be accidental, I've changed the casing a bit in my fork.

File: src/Asm89/Stack/CorsService.php (hunk 1 above)
Comment (barryvdh) [inline src/Asm89/Stack/CorsService.php]: Yes

**Comment by asm89** [comment]
Hm, you lower in the check itself?

https://github.com/barryvdh/laravel-cors/blob/61356cc82e87dc81605f1fcb9956b7877b11ec26/src/CorsService.php#L145

**Comment by barryvdh** [comment]
Gonne close this, as you are right that just array is probably enough.

**Pull request closed without merging**